qbft/controller.go

```diff
@@ -211,3 +208,7 @@ func (c *Controller) broadcastDecided(aggregatedCommit *SignedMessage) error {
 func (c *Controller) GetConfig() IConfig {
 	return c.config
 }
+
+func (c *Controller) SetIdentifier(identifier []byte) {
+	c.Identifier = identifier
```

Comment:
I can't understand how is this related to this PR

Reply:
Maybe if it isn't used in spec it can be deleted.
Implementation can add a simple setter as needed even if it isn't in spec